src/plugins/data/server/search/search_service.ts

```diff
@@ -316,7 +316,11 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
         strategy: strategyName,
         isStored: true,
       };
-      this.cancel(deps, searchId, searchOptions);
+      try {
+        await this.cancel(deps, searchId, searchOptions);
```

Comment:
nit: maybe we want to fire all cancel requests in parallel to speed this call-up? (same for extend below)